components/Card.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+export default function Card(props) {
+    return (
+        <View style={styles.card}>
+            <View style={styles.cardContent}>
+                { props.children }
+            </View>
+        </View>
+    );
+};
```

Comment:
I noticed you did some props destructuring in some components and this one you didn't 

```suggestion
export default function Card({children}) {
    return (
        <View style={styles.card}>
            <View style={styles.cardContent}>
                { children }
            </View>
        </View>
    );
};
```

Aside from specifying types with typescript or using the PropTypes library, this is a good way to help your IDE figure out what properties your component has available. 

> Learn more: [Destructuring in JS](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)